packages/ferric/src/template.ts

```diff
@@ -0,0 +1,58 @@
+export function cargoTomlTemplate(projectName: string) {
```

Comment:
May be preferable to have the `Cargo.toml` template file as a standalone file that the CLI reads in, so that we get proper syntax highlighting. It can have some template strings like `"PROJECT_NAME_REPLACE_ME"`.